administrator/components/com_localise/language/en-GB/com_localise.ini

```diff
@@ -36,6 +36,12 @@ ADMINISTRATOR="Administrator"
 SITE="Site"
 INSTALLATION="Installation"
 
+[Ajax]
+
+COM_LOCALISE_UPDATE_TRANSLATIONS_LIST_TASK_FLASH_ERROR="Error updating Translations list"
+COM_LOCALISE_UPDATE_TRANSLATIONS_LIST_TASK_FLASH_SUCCESS="Translations list has been updated"
+COM_LOCALISE_UPDATE_TRANSLATIONS_LIST_TASK_SUCCESS="Translations list has been updated."
```

Comment:
Do we really need 2 strings with the same value?

Reply:
I think yes due the "flash" ones should be a short text easy to read fast due there is only 3 seconds to read it.
The not "flash" can be longuer text due are the "system messages", displayed at the top of the page.

The text now is similar due no time to improve it yesterday, now will come updated at next PR